components/ldap.rst

```diff
@@ -35,38 +35,95 @@ using the following options:
 ``version``
     The version of the LDAP protocol to use
 
-``useSsl``
-    Whether or not to secure the connection using SSL
+``encryption``
+    Your encryption mechanism (``ssl``, ``tls`` or ``none``)
 
-``useStartTls``
-    Whether or not to secure the connection using StartTLS
+``connection_string``
```

Comment:
Unrelated to this PR, but this option name looks weird. Is there a better alternative to `connection_string` ? For example, like `dsn` in the database world.